app/views/sponsors/show.html.haml

```diff
@@ -0,0 +1,12 @@
+%div.sponsors-page-wrapper
+  %h3
+    Sponsors
+
+  #sponsors
+    - @sponsors_by_tier.each do |tier, sponsors|
+      .sponsor-tier-wrapper
+        .tier-title
+          = tier
+        .sponsors-wrapper
+          - sponsors.each do |sponsor|
```

Comment:
Rails might have a short cut here as well: 

```
= render sponsors
```

see https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections and https://thoughtbot.com/blog/rendering-collections-in-rails

Reply:
Witchcraft! (I updated this)